src/views/AppRoot/component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { BrowserRouter, Route, Redirect } from 'react-router-dom'
+
+import LoginScreen from '../pages/Login'
+import Dashboard from '../pages/Dashboard'
+import StubsRoot from '../components/stubs/StubsRoot'
+
+const AppRootComponent = ({ isLogged }) => (
+  <BrowserRouter>
+    <Route
+      path="/login"
+      component={LoginScreen}
+    />
+    <Route
+      path="/"
+      component={Dashboard}
+    />
+    <Route
+      path="/stubs"
+      component={Dashboard}
+    />
+    {!isLogged && <Redirect to="/login" />}
```

Comment:
create component like `PrivateRoute` which will check `isLoggedIn` variable internally. 
```
const PrivateRoute = ({ component: Component, isLoggedIn, ...props }) => (
  <Route
    {...props}
    render={(props) => isLoggedIn
      ? (
        <PrivateLayout>
          <Component {...props } />
        </PrivateLayout>
      ) : (
        <Redirect to="/login" />
      )
    }
  />
)
```

Reply:
you also need something like `GuestRoute` component to forbid access to signup and login pages for authorized users